stream-chat-android-ui-components/src/main/res/values/attrs.xml

```diff
@@ -4,13 +4,14 @@
     <attr name="streamUiReactionsBubbleColorTheirs" format="color|reference" />
 
     <!--Themes attributes-->
-    <declare-styleable name="StreamUiStyleable" >
+    <declare-styleable name="StreamUiStyleable">
         <attr name="streamUiValidTheme" format="boolean" />
         <attr name="streamUiTheme" format="reference" />
         <attr name="streamUiMessageListItemAvatarStyle" format="reference" />
         <attr name="streamUiChannelListItemAvatarStyle" format="reference" />
         <attr name="streamUiGlobalAvatarBorderColor" format="reference|color" />
         <attr name="streamUiGlobalAvatarTextColor" format="reference|color" />
         <attr name="streamUiGlobalAvatarOnlineIndicatorColor" format="reference|color" />
+        <attr name="streamUiChannelActionsDialogStyle" format="reference" />
     </declare-styleable>
```

Comment:
Make `streamUiChannelActionsDialogStyle` attribute to reference an overriden style to override channel actions style globally